edward/stats/distributions.py

```diff
@@ -9,114 +9,83 @@ class Distribution:
     """Template for all distributions."""
```

Comment:
Given that we now have API docs, can we delete this class altogether and add anything it offers manually within the doc pages?